common/json_stream.c

```diff
@@ -370,11 +370,11 @@ void json_add_timeabs(struct json_stream *result, const char *fieldname,
 		      struct timeabs t)
 {
 	json_add_primitive_fmt(result, fieldname,
-			       "%" PRIu64 ".%03" PRIu64,
```

Comment:
Commit message is a bit wrong:

```
Timeabs only printed in milliseconds and json add time outputs a string which is weird
```

Should be "json_add_timeabs only printed.... and json_add_time outputs..."